docs/src/documentation/03-components/01-action/buttonmobilestore/03-accessibility.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/buttonmobilestore/accessibility/
+---
+
+## Accessibility
+
+The ButtonMobileStore component has been designed with accessibility in mind, providing accessible mobile app store download buttons with proper semantic structure and screen reader support.
+
+### Accessibility Props
+
+The following props are available to improve the accessibility of your ButtonMobileStore component:
+
+| Name            | Type      | Description                                                                                                                                                                                                                             |
+| :-------------- | :-------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| alt             | `string`  | **Required.** Provides alternative text description of the store button image for screen readers. Should clearly describe the action and store type (e.g., "Download on the App Store" or "Get it on Google Play"). Must be translated. |
+| title           | `string`  | Provides additional context as a tooltip and is announced by screen readers after the alt text. Use to provide supplementary information about the app or download. Must be translated.                                                 |
+| stopPropagation | `boolean` | Controls event bubbling behavior. When `true`, prevents the click event from bubbling up to parent elements, which can help avoid focus management issues in complex layouts.                                                           |
+
+### Automatic Accessibility Features
+
+- The component automatically renders as a semantic `<a>` element, creating a proper link that assistive technologies can identify and navigate
```

Comment:
I wonder if we should make some adjustments to the component because currently when `href` is not provided but `onClick` is provided, we still render `<a>` but it's not keyboard focusable.

Reply:
Looks good.